test/time_tracker/timers/pubsub_test.clj

```diff
@@ -1,6 +1,7 @@
 (ns time-tracker.timers.pubsub-test
   (:require [clojure.test :refer :all]
             [clojure.core.async :refer [chan alt!! put!] :as async]
+            [clojure.string :as str]
```

Comment:
@deobald since `spec` was introduced, it has become common to require `clojure.spec :as s`.